lib/request/delete.js

```diff
@@ -95,6 +95,9 @@ module.exports = function (context) {
 
         for (k = 0, l = links.length; k < l; k++) {
           field = links[k]
+
+          if (!record[field]) continue
+
           inverseField = fields[field][inverseKey]
 
           if (!record.hasOwnProperty(field) || !inverseField) continue
```

Comment:
I think it would be better to tweak this instead of `hasOwnProperty` to check `!(field in record)`

Reply:
I'm not sure I follow `!(field in record)`?

How about:
```js
          if (!record.hasOwnProperty(field)) continue

          inverseField = fields[field][inverseKey]

          if (!inverseField) continue
```

At this point in the night after a long day I might be way off :)